src/components/DigitalTerrainWebApp.jsx

```diff
@@ -1,26 +1,17 @@
 import React from 'react';
-import logo from '../logo.svg';
 import './DigitalTerrainWebApp.css';
+import 'antd/dist/antd.css';
+import DigitalTerrainWebContainer from './DigitalTerrainWebContainer';
 
-function DigitalTerrainWebApp() {
-  return (
-    <div className="digital-terrain-web-app">
-      <header className="digital-terrain-web-app-header">
-        <img src={logo} className="digital-terrain-web-app-logo" alt="logo" />
-        <p>
-          Edit <code>src/components/DigitalTerrainWebbApp.jsx</code> and save to reload yo.
-        </p>
-        <a
-          className="digital-terrain-web-app-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class DigitalTerrainWebApp extends React.Component {
+  render() {
+    return (
+      <div className="digital-terrain-web-app">
+        <DigitalTerrainWebContainer />
+      </div>
+    );
+  }
```

Comment:
nit / question: Since you are not using state why not make the linter happy?

Reply:
Good question. I thought I could recall an error because there wasn't a component that extended React.Component but I just tried making it a functional one instead and it worked without issue. So I reverted it and removed the eslint-disable.

Reply:
+1